pkgs/by-name/ra/radiance/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  cmake,
+  fetchFromGitHub,
+  runCommand,
+  lib,
+  libGLU,
+  stdenv,
+  libX11,
+  tcsh,
+  tk,
+}:
+let
+  csh = runCommand "csh" { } ''
+    mkdir -p $out/bin
+    cp ${tcsh}/bin/tcsh $out/bin/csh
```

Comment:
```suggestion
    ln -s ${tcsh}/bin/tcsh $out/bin/csh
```

Reply:
I'd like to recommend use of `${lib.getExe tcsh}` (or `${lib.getExe' tcsh "csh"}` if the tcsh package does not define `meta.mainProgram`) instead of `${tcsh}/bin/csh`